src/reynir_correct/errtokenizer.py

```diff
@@ -1749,7 +1770,7 @@ def only_suggest(token: CorrectToken, m: Sequence[BIN_Tuple]) -> bool:
                 context=tuple(context[-3:-1]),
                 at_sentence_start=at_sentence_start,
             )
-            if corrected_txt != token.txt:
+            if corrected_txt != token.txt and len(corrected_txt) > 1 and len(token.txt) > 1:
```

Comment:
This never corrects single letters, and never corrects multi-letter words to single letters. Is that definitely intentional?